dev-haskell/cabal/cabal-3.8.1.0.ebuild

```
@@ -0,0 +1,32 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.7.2.2.9999
+#hackport: flags: -parsec-struct-diff
+
+CABAL_PN="Cabal"
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+CABAL_FEATURES+=" nocabaldep" # in case installed Cabal is broken
+inherit haskell-cabal
+
+DESCRIPTION="A framework for packaging Haskell software"
+HOMEPAGE="https://www.haskell.org/cabal/"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND=">=dev-lang/ghc-8.4.3:=
+"
```

```suggestion
RDEPEND=">=dev-lang/ghc-8.4.3:=
	>=dev-haskell/cabal-syntax-3.8 <dev-haskell/cabal-syntax-3.9
"
```

- [`Cabal-3.8.1.0/Cabal.cabal`](https://hackage.haskell.org/package/Cabal-3.8.1.0/Cabal.cabal)
  ```cabal
    build-depends:
      Cabal-syntax >= 3.8    && < 3.9,
  ```
```cabal

---

It looks like we will either need to make `process` an upgradable package or depend on `>=dev-lang/ghc-9.4`:

- [`Cabal-3.8.1.0/Cabal.cabal`](https://hackage.haskell.org/package/Cabal-3.8.1.0/Cabal.cabal)
  ```cabal
    -- pull in process version with fixed waitForProcess error
    if impl(ghc >=8.2)
      build-depends: process >= 1.6.14.0
  ```